### PR TITLE
CSI certification: check if test is running against OCP cluster

### PR DESCRIPTION
**Pull request by bertinatto**
CC @openshift/storage

### CODE REVIEW

**Comment by bertinatto** [comment]
@openshift/storage PTAL

**Comment by bertinatto** [comment]
Test result: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/24491/pull-ci-openshift-origin-master-e2e-aws-csi/2092

File: test/extended/csi/certification.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+		g.By("checking for the Cluster Version Operator")
+		waitForCVO(coreclient.CoreV1().Namespaces())
+
+		cv, err := c.ConfigV1().ClusterVersions().Get(context.TODO(), "version", metav1.GetOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		clusterVersion, err := json.Marshal(cv)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		e2e.Logf("Detected cluster version: %s", clusterVersion)
```
Comment (jsafrane) [inline test/extended/csi/certification.go:38]: is this ever printed to openshift-tests stdout?

Comment (bertinatto) [reply test/extended/csi/certification.go]: It's not, removed

**Comment by bertinatto** [comment]
/assign @smarterclayton 
Clayton, there's an ask from the CSI Certification team to make sure the test suite is running in an OpenShift cluster. The way I did it was making sure the CVO is running. Could you take a look at this and let me knnow if it makes sense?

**Pull request closed without merging**